File: Views/src/app/_components/administrative/local/services/service-budget/dto/service-budget-dto.ts
import { ClientDto } from "src/app/_components/administrative/client/dto/client-dto";
import { EquipamentDto } from "../../items-services/dto/equipament-dto";
import { SolutionPriceDto } from "./solution-price-dto";

export class ServiceBudgetDto {
  id: number;
  client: ClientDto;
  clientId: number;
  entryDate: Date;
  entryDateOs: Date;
  clientProblems: string;
  status: string;
  visually: string;
  osMake: boolean;
  finished: boolean;
  solutionsPrices: SolutionPriceDto[]
}

File: Services/Dto/ServiceBudgetDto.cs
using System;
using System.Collections.Generic;

namespace Services.Dto

{
    public class ServiceBudgetDto
    {
        public int Id { get; set; }
        public int ClientId { get; set; }
        public ClientDto Client { get; set; }
        public string ClientNoRegister { get; set; }
        public string ClientProblems { get; set; }
        public string Status { get; set; }
        public string Visually { get; set; }
        public DateTime EntryDate { get; set; }
        public DateTime EntryDateOs{ get; set; }
        public bool Finished { get; set; }
        public bool OsMake {get; set;}
        public List<SolutionPriceDto> SolutionsPrices { get; set; }

    }
}
